docker-compose.yaml

```diff
@@ -4,7 +4,7 @@ services:
 
   pmm-server:
     container_name: pmm-server
-    image: perconalab/pmm-server-fb:latest
+    image: perconalab/pmm-server-fb:PR-169-aa82c83
```

Comment:
Why?

Reply:
Mistakenly added. And `:latest` doesn't exist.

Reply:
`dev-latest`?